compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -870,6 +878,16 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
       pre_trans->a(pred_node);
       node->x(pre_trans);
     }
+    else if ((not is_const(node->x())) and (not is_const(node->y())))
```

Comment:
Current codes look like this.

```cxx
    if (is_NCHW_with_s_const<luci::CircleMaximum>(node, pred_node, comp_constant))
    {
      auto pre_trans = create_pre_transpose(node);
      pre_trans->a(pred_node);
      node->x(pre_trans);
    }
    else if ((not is_const(node->x())) and (not is_const(node->y())))
    {
      auto pre_trans_x = create_pre_transpose(node);
      pre_trans_x->a(node->x());
      node->x(pre_trans_x);

      auto pre_trans_y = create_pre_transpose(node);
      pre_trans_y->a(node->y());
      node->y(pre_trans_y);
    }
    else
    {
      // TODO support other cases
      return false;
    }
```

Seems considering only y const case is missing. How about this logic instead?
```cxx
    if (is_const(node->x())
    {
      auto pre_trans = create_pre_transpose(node);
      pre_trans->a(pred_node);
      node->x(pre_trans);
    }
    if (is_const(node->y())
    {
      auto pre_trans_y = create_pre_transpose(node);
      pre_trans_y->a(node->y());
      node->y(pre_trans_y);
    }
```

Of course you should also revisit `is_NCHW_with_s_const` method.

Reply:
> Seems considering only y const case is missing

The case is handled by `is_NCHW_with_s_const`.

Reply:
Ah, I mean the case y isn't const case. It always call just `x` method. Is it right?

Reply:
(optional) It'd be better to rename `is_NCHW_with_s_const` methdo to `set_NCHW_with_s_const` because "is" prefix doesn't look like it is going to setting something.

Reply:
You're right. node->y() should be called if y is non-const. Existing code should be updated. Thanks for finding.

Reply:
Let me fix it in another PR :)